### DISCUSSION TITLE
Auto-sleep after reaching charge limit

### DISCUSSION THREAD

**Discussion by sophlzy3** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#693] [Q&A] [answered]
For the context, this started happening after I changed my MacBook Pro battery from a non-Apple manufacturer.

Every time I have AlDente open on my computer, my computer sleeps minutes after the battery has reached the designated limit set in the app (sometimes 20min, sometimes 15min, sometimes 5min).
Also, sometimes when the computer has slept for too long, it automatically shuts down.

Before I replaced the battery this did not happen. 

Is there any solution to this or should I start an issue?

**Comment by Bachsau** [comment, marked as answer]
If you didn't change anything else, that might be an issue with your battery. Third party batteries are almost always crap.

**Reply by sophlzy3** [reply]
Ohhhh that's probably it. I tried resetting both PAM and SMC but still it doesn't work. 
 Thank you!
